SECURITY_ja.md

```diff
@@ -0,0 +1,63 @@
+# PaddlePaddle を安全に使用する
```

Comment:
Although I can understand your Japanese translation to the security documents, my Japanese is not good enough to give review comments. 

But I think we can accept your translation, this is a great addition to PaddlePaddle community.

I will merge your PR, after you remove the translation to Code of Conduct.

Reply:
@jzhang533 
Thank you for your confirmation.
Code of Conduct file has been removed.